platform/ui/src/components/AllInOneMenu/__stories__/allInOneMenu.stories.mdx

```diff
@@ -0,0 +1,137 @@
+import AllInOneMenu from '../AllInOneMenu';
+import AllInOneMenuItemPanel from '../AllInOneMenuItemPanel';
+import AllInOneMenuSubMenu from '../AllInOneMenuSubMenu';
+import AllInOneMenuItem from '../AllInOneMenuItem';
+import AllInOneMenuDividerItem from '../AllInOneMenuDividerItem';
+import AllInOneMenuHeaderItem from '../AllInOneMenuHeaderItem';
+import InputRange from '../../InputRange/index.js';
+import SwitchButton from '../../SwitchButton/index.js';
+
+import { ArgsTable, Story, Canvas, Meta } from '@storybook/addon-docs';
+import { createComponentTemplate } from '../../../storybook/functions/create-component-story';
+
+export const argTypes = {
+  component: AllInOneMenu,
+  title: 'Components/AllInOneMenu',
+};
+
+<Meta
+  title="Components/AllInOneMenu"
+  component={AllInOneMenu}
+/>
+
+export const AllInOneMenuTemplate = args => (
+  <div className="w-80">
+    <AllInOneMenu {...args}>
+      <AllInOneMenuItemPanel label="Panel 1">
+        <AllInOneMenuItem
+          label="Item 1"
+          key="0"
+          onItemClick={() => console.info('Item 1 clicked.')}
+        ></AllInOneMenuItem>
+        <AllInOneMenuItem
+          label="Item 2"
+          alternateLabel="Alt item 2"
+          key="1"
+        ></AllInOneMenuItem>
+        <div
+          className="all-in-one-menu-item my-1 flex w-full justify-center"
+          style={{ height: '50px' }}
+        >
```

Comment:
can we extract these styles and put it as an enum, to use them elsewhere for our custom sub components?

Reply:
Yes, well the intent was for anyone interested to simply use the `all-in-one-menu-item` class. I believe that is better than have a const or enum of styles. I cleaned up the code so that the custom component you see here only has the one className. Of course, because it is a custom component there will always be custom styling like the height attribute you see here.